docs/ref/checks.txt

```diff
@@ -234,6 +234,9 @@ Model fields
 * **fields.W907**: ``django.contrib.postgres.fields.CITextField`` is
   deprecated. Support for it (except in historical migrations) will be removed
   in Django 5.1.
+* **fields.E220**: ``<database>`` does not support ``ComputedField``\s.
+* **fields.E221**: ``<database>`` does not support ``ComputedField``\s with constant parameters.
+* **fields.E222**: ``<database>`` does not support ``ComputedField``\s with no persistence.
```

Comment:
This is my first P/R, so I am not sure what the protocol is for adding new error checks. Any advise would be greatly appreciated!